### PR TITLE
nvs@1.6.0: Use call operator instead of powershell cli

### LINKED ISSUE #3350: nvs post install throws an error
## Bug Report

**Package Name:** nvs

### Current Behaviour

The post install script throws an error:
```pwsh
❯ scoop install nvs
Installing 'nvs' (1.6.0) [64bit]
Loading nvs-1.6.0.msi from cache
Checking hash of nvs-1.6.0.msi ... ok.
Extracting nvs-1.6.0.msi ... done.
Linking ~\scoop\apps\nvs\current => ~\scoop\apps\nvs\1.6.0
Persisting nodejs
Running post-install script...
Get-Content:
Line |
   2 |  $CONF = (Get-Content "$persist_dir\nodejs\settings.json") | ConvertFr …
     |           ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
     | Cannot find path 'C:\Users\Segev\scoop\persist\nvs\nodejs\settings.json' because it does not exist.
Add-Member:
Line |
   3 |  … Name linkToSystem -NotePropertyValue $false -Force -InputObject $CONF
     |                                                                    ~~~~~
     | Cannot bind argument to parameter 'InputObject' because it is null.
'nvs' (1.6.0) was installed successfully!
```

### Expected Behaviour

It shouldn't throw an error.

### Possible Solution

Perhaps adding`-ea SilentlyContinue` to `Get-Content` and `if (-not $?) { $CONF = New-Object -TypeName PSObject }

Why is setting `linkToSystem` required, and is it desired to do so or an issue with nvs? (I guess it detects the scoop installation directory thinking that it's a system wide installation of nvs?)

### System details

**Windows version:** 10

**OS architecture:** 64bit

**PowerShell version:** 7.2.1

**Additional software:** Git, starship prompt

#### Scoop Configuration

```json
{
  "lastupdate": "2022-03-04T16:28:15.9629229+02:00",
  "SCOOP_REPO": "https://github.com/lukesampson/scoop",
  "SCOOP_BRANCH": "master"
}
```

### PR DESCRIPTION
**Pull request by segevfiner**
Closes #3350

- [x] I have read the [Contributing Guide](https://github.com/ScoopInstaller/.github/blob/main/.github/CONTRIBUTING.md).

### CODE REVIEW

**Comment by segevfiner** [comment]
Derp something in Git got messed up, rebasing.

**Comment by segevfiner** [comment]
@rashil2000

File: bucket/nvs.json [hunk 1]
```diff
@@ -12,7 +12,7 @@
         "NVS_HOME": "$dir\\nodejs"
     },
     "post_install": [
-        "powershell -NoProfile -File $dir\\nvs.ps1 remote | Out-Null",
+        "& $dir\\nvs.ps1 remote | Out-Null",
```
Comment (rashil2000) [inline bucket/nvs.json:15]:
```suggestion
        "& \"$dir\\nvs.ps1\" remote | Out-Null",
```

**Pull request merged**